provider/src/main/kotlin/org/gradle/kotlin/dsl/provider/KotlinBuildScriptCompiler.kt

```diff
@@ -197,7 +197,7 @@ class KotlinBuildScriptCompiler(
 
         val (lineNumber, compiledScript) = compiledPluginsBlock
         val pluginsBlockClass = classFrom(compiledScript, baseScope.createChild("plugins"))
-        val pluginDependenciesSpec = pluginRequestCollector.createSpec(lineNumber)
+        val pluginDependenciesSpec = pluginRequestCollector.createPluginDependencySpec(lineNumber)
```

Comment:
:o: I understand this is coming from the Gradle API but shouldn't it be called `createPluginDependenciesSpec`?

Reply:
That's right! Let me fix this and I'll get back to you for another pass on this review.

Reply:
@bamboo fixed